packages/config/config/devices/0x017f/wnk-mot1.json

```diff
@@ -0,0 +1,192 @@
+{
+	"manufacturer": "Wink",
+	"manufacturerId": "0x017f",
+	"label": "WNK-MOT1",
+	"description": "Motion Sensor",
+	"devices": [
+		{
+			"productType": "0x0101",
+			"productId": "0x0001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "LifeLine",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "BASIC_SET",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "NOTIFICATION_REPORT",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "SENSOR_BINARY_REPORT",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Sensitivity",
+			"description": "Motion sensor sensitivity Level. Valid values 8 to 255.",
+			"valueSize": 1,
+			"minValue": 8,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Motion Cleared Time Delay",
+			"description": "Valid values 5 to 600 seconds.",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 5,
+			"maxValue": 600,
+			"defaultValue": 600,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Basic Set Level",
+			"description": "0: Turn Off // 1 to 99: Dim Level // 255: On",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "Enable/Disable Motion Detection",
+			"description": "",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"5": {
+			"label": "Group 2 Ambient Light Threshold",
+			"description": "Ambient Light Threshold for the Group 2 Association trigger.  Vaid values 5 to 1000 lux.",
+			"valueSize": 2,
+			"unit": "lux",
+			"minValue": 5,
+			"maxValue": 1000,
+			"defaultValue": 1000,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"6": {
+			"label": "Retrigger Interval",
+			"description": "Prevent retriggering within the specified time period.  Valid values 1 to 8 seconds.",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 8,
+			"defaultValue": 8,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"7": {
+			"label": "Light Sensing Interval",
+			"description": "Valid values 60 to 3600 seconds.",
```

Comment:
Remove this description - it doesn't add any value over the configured value ranges